exercism/wordy/wordy.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  WordProblem: class WordProblem {
+    constructor(input){
+      this.input = input,
+      this.operands = {
+        'plus': '+',
+        'minus': '-',
+        'multiplied': '*',
+        'divided': '/'
+      },
+      this.sequence = []
+
+    }
+
+    answer() {
+      let splitString = this.input.replace('?',"")
+      .replace(/by/g,'').split(' ').slice(2)
+      let operandsKeys = Object.keys(this.operands)
```

Comment:
This code is a little difficult to read since there is no whitespace to separate the different logical sections.  I encourage you to use whitespace to help guide the reader through the logic.  Also, avoid the use of one time, or temp, variables, and just use the value inline.